Source/CloudyPlayerManager/Private/CloudyPlayerManager.cpp

```diff
@@ -86,11 +86,19 @@ bool CCloudyPlayerManagerModule::RemovePlayer(int32 ControllerId, int32 GameSess
 	ULocalPlayer* const ExistingPlayer = GameInstance->FindLocalPlayerFromControllerId(ControllerId);
 	bool Success = false;
 
+
+	
+	
+	
+
+
```

Comment:
eh? sorry was in a hurry. cos of time issues and importance, I'll only fix this if there are other issues.